ui/src/app/pages/profile/demographic-survey.tsx

```diff
@@ -101,7 +118,16 @@ export class DemographicSurvey extends React.Component<Props, State> {
   }
 
   validateDemographicSurvey(demographicSurvey) {
+    validate.validators.nullBoolean = v => (v === true || v === false || v === null) ? undefined : 'value must be selected'
     const validationCheck = {
+      race: { presence: { allowEmpty: false } },
```

Comment:
Is "...must not be blank" the default error message for `allowEmpty: false` ?  I'd prefer a custom message.  Something like "must not be blank.  You may select 'Prefer not to answer' to continue"

Reply:
I added 

> You may select "Prefer not to answer" for each unfilled item to continue
 To the end to prevent noise / redundancy in the message